content/vendor/guides/production-ready.md

```diff
@@ -0,0 +1,194 @@
+
+If you expect to also install stateful services into existing clusters, you'll likely want to expose [preflight analyzers that check for the existence of a storage class](https://troubleshoot.sh/reference/analyzers/storage-class/).
+
+
+### Namespaces
+
+It is *strongly* advised that applications be architected to deploy a single application into a single namespace when possible. This will give the most flexibility when deploying to end user environments. Don't specify a namespace in your YAML resources, or try to make this user-configurable using the `kots.io` `Config` object, just leave namespace blank. Letting the end user manage namespaces will be the easiest way to reduce friction. The ability to manage namespaces in kotsadm will be added in an upcoming KOTS release, so if you intend to  deploy multiple apps using a single Admin Console, it is be reasonable to architect your app assuming the end user will manage the Namespaces that each component runs in.
+
+### Helm
+
+Helm charts are supported by KOTS but not required. If for applications that are already packaged using helm, then the helm support in KOTS can help get an app packaged faster. If an application does not presently use helm, there's no requirement to use helm, as the KOTS built-in templating includes much of the same functionality, and the Admin Console includes a deep [kustomize.io integration](/vendor/kotsadm/updating/patching-with-kustomize) to greatly reduce the amount of templating required by app maintainers in the first place.
+
+### Operators
+Operators are good for specific use cases, we've written in-depth about them in our [Operators Blog Post](https://blog.replicated.com/operators-in-kots/). In general we recommend thinking deeply about the problem space an application solves before going down the operator path. They're really cool and powerful, but take a lot of time to build and maintain
+
+
+<!-- coming soon, wait for guide
+### Licensing and Entitlements
+  - how we think about it 
+  - basic enforcement
+  - advanced enforcement via API
+
+Basic license enforcement comes baked in for free -- expiration 
+
+Our recommendation is always to enforce licensing with a strong contract, but if you have strict licensing needs, you can always implement [Runtime License Validation](/vendor/guides/runtime-license-validation) in your application. This involves modifying your application code to query an API exposed by KOTS to consume the license payload, and validating the signature in the hot code paths in your application's core business logic.
+-->
+
+### Bundling and Analyzing Logs with Support bundle
+
+A robust support bundle is essential to minimizing back-and-forth when things go wrong. At a very minimum every app's support bundle should contain logs for an application's core pods. Usually this will be done with label selectors. To the labels for an application, either inspect the YAML, or run
```

Comment:
```suggestion
A robust support bundle is essential to minimizing back-and-forth when things go wrong. At a very minimum every app's support bundle should contain logs for an application's core pods. Usually this will be done with label selectors. To get the labels for an application, either inspect the YAML, or run
```